### PR TITLE
Feature/scale ranges step size

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

File: app/ts/Subproblem/SubproblemButtons/AddSubproblemButton/AddSubproblemScaleRanges/ScalesSlider/ScalesSlider.tsx [hunk 1]
```diff
@@ -160,6 +184,17 @@ export default function ScalesSlider({criterion}: {criterion: ICriterion}) {
...
+        Step size:{' '}
+        <Select
+          native
+          id={`step-size-selector-${criterion.id}`}
+          value={stepSize}
+          onChange={handleStepSizeChange}
+        >
+          {renderStepSizeOptions()}
+        </Select>
+      </Grid>
```
Comment (chinook25) [inline app/ts/Subproblem/SubproblemButtons/AddSubproblemButton/AddSubproblemScaleRanges/ScalesSlider/ScalesSlider.tsx:197]: Consider making this its own component.

File: test/endToEnd/subproblemTest.js [hunk 2]
```diff
@@ -144,16 +135,23 @@ function changeScale(browser) {
...
     .mouseButtonUp(0)
     .assert.containsText(upperValueLabel, '500')
+    .assert.containsText(stepSizeSelector, '10')
+    .click(stepSizeSelector)
+    .assert.containsText(stepSizeSelector + '/option[1]', '100')
+    .assert.containsText(stepSizeSelector + '/option[2]', '10')
+    .assert.containsText(stepSizeSelector + '/option[3]', '1')
+    .click(stepSizeSelector + '/option[3]')
+    .assert.containsText(stepSizeSelector, '1')
     .useCss()
```
Comment (chinook25) [inline test/endToEnd/subproblemTest.js:155]: If we change the step size, shouldn't we also check if the values get updated correctly?

**Review by chinook25 — COMMENTED** [review]
minor things

File: test/endToEnd/subproblemTest.js (hunk 2 above)
Comment (kzalite) [reply test/endToEnd/subproblemTest.js:155]: Can't get it to work in headless mode

**Pull request merged**